app/src/main/java/com/example/parkingsystem/mvp/contract/ParkingReservationContract.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.parkingsystem.mvp.contract
+
+import android.app.DatePickerDialog
+import android.app.TimePickerDialog
+import java.util.*
```

Comment:
Pls not use wildcards, configure in settings and remove the use of wildcads

Reply:
still here

Reply:
Done